src/components/Header.js

```diff
@@ -1,28 +1,28 @@
-import React from "react"
+import React from 'react';
 
 const Header = () => {
   const headerStyle = {
-    padding: "20px 0",
-    lineHeight: "1.5em",
-  }
+    padding: '20px 0',
+    lineHeight: '1.5em',
+  };
```

Comment:
Again for sake of clarity of your code, you should try as much as possible to keep your styles in css/sass files.